### ISSUE TITLE
Error while opening wsf files

### ISSUE THREAD

**Issue by rm-NoobInCoding** [Foxxyyy/CodeX.Games.RDR1#3]
![image_2024-10-30_172835957](https://github.com/user-attachments/assets/6045bf93-f9d1-4acd-bf07-bf7cf7ce04e9)

**Comment by Foxxyyy** [comment]
@rm-NoobInCoding 

Yep, hopefully I can fix that later! These WSF files cannot be viewed yet.

For context, Double Eleven used a 3rd party library called Crunch for transcoding DDS textures to CRN textures for the UI.
Same as the release of the Nintendo Switch/PS4 versions.
I've sadly never been able yet to decode these properly.

**Comment by rm-NoobInCoding** [comment]
I'm trying to get the font mapping from the list.wsf file. It would be great if you could provide the structure or at least edit the code to read part of blocks!

**Comment by rm-NoobInCoding** [comment]
I did some research about font blocks in wsf files. I sent you a discord friend request for more talk about wsf.

**Comment by Nanoposan69** [comment]
On PC, D11 switch back texture format to DXT for almost all flash files. Crunch now only used in sharedflash.wdt and effects.wsf
![image](https://github.com/user-attachments/assets/6a23f907-443f-47e1-aade-56eb6c7faf38)
font atlas stored in wsf, but textures now overdrives from SDF folder.

PS. You can contact me in discord (nanoposan69) and we can talk more about formats + we made text tools (strtbl and wst/ost/sst, cst, xst)

**Comment by rm-NoobInCoding** [comment]
The problem is all wsf files are stored as resource in RPF files. and the library can only import files as raw. so this is big problem.

**Comment by Foxxyyy** [comment]
Ohh this is great @Nanoposan69 I'll DM you on Discord
